packages/did/src/Did.chain.ts

```diff
@@ -387,21 +391,20 @@ interface IDidAuthorizedCallOperation extends Struct {
 
 // ### EXTRINSICS
 
-type FormattedPublicKey<K extends NewDidKey> = TypedValue<
-  Capitalize<K['type']>,
-  K['publicKey']
->
+export function encodePublicKey(
+  key: NewDidVerificationKey
+): EncodedVerificationKey
+export function encodePublicKey(key: NewDidEncryptionKey): EncodedEncryptionKey
+
 /**
  * Transforms a DID public key record to an enum-type key-value pair required in many key-related extrinsics.
  *
  * @param key Object describing data associated with a public key.
  * @returns Data restructured to allow SCALE encoding by polkadot api.
  */
-export function formatPublicKey<K extends NewDidKey>(
-  key: K
-): FormattedPublicKey<K> {
+export function encodePublicKey(key: NewDidKey): EncodedKey {
```

Comment:
I think this is a good moment to offer this renaming. Our naming convention for preparing some data for the blockchain is "encode", and it should be used here as well. This renaming is responsible for the most of the diff here.

Reply:
Not a strong opinion, but I think it doesn't hurt to use that function in this file...